pkg/controller/cheinstallation/che.go

```diff
@@ -79,6 +107,43 @@ func NewOperatorGroup(ns string) *olmv1.OperatorGroup {
 	}
 }
 
+func NewCheCluster(ns string) *orgv1.CheCluster {
+	return &orgv1.CheCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      CheClusterName,
+			Namespace: ns,
+			Labels:    toolchain.Labels(),
+		},
+
+		Spec: orgv1.CheClusterSpec{
+			Server: orgv1.CheClusterSpecServer{
+				CheFlavor:      CheFlavorName,
+				TlsSupport:     true,
+				SelfSignedCert: true,
```

Comment:
Let's set both TlsSupport and SelfSingedCert to false. We will lose the link to Che from DevConsole but otherwise the users will need to deal with go-to-all-roots-and-accept-certs hell. It also probably not right to enable self signed certificates by default anyway (and we can't set TLS to true by default because of that).

cc: @xcoulon

Reply:
Done https://github.com/codeready-toolchain/toolchain-operator/pull/17/commits/02c350a6e0d9c6365d30f0cecab9550189d7e51c

Check discussion at - https://coreos.slack.com/archives/CJP2KF717/p1575962883045100